proto/umee/leverage/v1/inspector.proto

```diff
@@ -0,0 +1,140 @@
+message QueryInspectResponse {
+  repeated BorrowerSummary borrowers = 1 [
+    (gogoproto.nullable) = false
+  ];
+}
+
+// QueryInspectNeatResponse defines the response structure for the InspectNeat gRPC service handler.
+message QueryInspectNeatResponse {
+  repeated BorrowerSummaryNeat borrowers = 1 [
+    (gogoproto.nullable) = false
+  ];
+}
+
+// QueryRiskDataResponse defines the response structure for the RiskData gRPC service handler.
+message QueryRiskDataResponse {
+  repeated AccountDecBalances borrowers = 1 [
+    (gogoproto.nullable) = false
+  ];
+}
+
+// BorrowerSummary defines a borrower's address and account health.
+message BorrowerSummary {
+  string address = 1;
+  // Supplied Value is the total USD value of all tokens the account has supplied, includng interest earned.
+  string supplied_value = 2 [
+    (gogoproto.customtype) = "github.com/cosmos/cosmos-sdk/types.Dec",
+    (gogoproto.nullable)   = false
+  ];
+  // Collateral Value is the total USD value of the account's collateral.
+  string collateral_value = 3 [
+    (gogoproto.customtype) = "github.com/cosmos/cosmos-sdk/types.Dec",
+    (gogoproto.nullable)   = false
+  ];
+  // Borrowed Value is the total USD value of all tokens the account has borrowed, including interest owed.
+  string borrowed_value = 4 [
+    (gogoproto.customtype) = "github.com/cosmos/cosmos-sdk/types.Dec",
+    (gogoproto.nullable)   = false
+  ];
+  // Borrow Limit is the maximum Borrowed Value the account is allowed to reach through direct borrowing.
+  string borrow_limit = 5 [
+    (gogoproto.customtype) = "github.com/cosmos/cosmos-sdk/types.Dec",
+    (gogoproto.nullable)   = false
+  ];
+  // Liquidation Threshold is the Borrowed Value at which the account becomes eligible for liquidation.
+  string liquidation_threshold = 6 [
+    (gogoproto.customtype) = "github.com/cosmos/cosmos-sdk/types.Dec",
+    (gogoproto.nullable)   = false
+  ];
+  // SpecificCollateralValue is the amount of Collateral Value which is the result of a specified collateral denom.
+  string specific_collateral_value = 7 [
+    (gogoproto.customtype) = "github.com/cosmos/cosmos-sdk/types.Dec",
+    (gogoproto.nullable)   = false
+  ];
+  // SpecificBorrowValue is the amount of Borrowed Value which is the result of a specified borrowed denom.
+  string specific_borrow_value = 8 [
+    (gogoproto.customtype) = "github.com/cosmos/cosmos-sdk/types.Dec",
+    (gogoproto.nullable)   = false
+  ];
+}
+
+// BorrowerSummaryNeat defines a borrower's address and account health without requiring sdk.Dec formatting
+message BorrowerSummaryNeat {
+  string Account = 1;
+  // Borrowed is account's borrowed value, either in total or just a specified denom.
+  double Borrowed = 2;
+  // LTB is borrowed value as a portion of borrow limit.
+  double LTB = 3;
+  // LTL is borrowed value as a portion of liquidation threshold.
+  double LTL = 4;
+  // LTV is the borrowed value as a portion of collateral value.
+  double LTV = 5;
```

Comment:
what `LT` stands for in the queries above?

Reply:
"loan to" - for example, LTV is "loan to value" and is a term that appears in the literature. 

The other two are not based on any official terms - they stand for "loan to borrow limit" and loan to liquidation threshold".

Not 100% informative as they are now, but if I give them verbose names like `LoanToValue` then the output of the query is crowded by the same words repeated over and over each entry, and the numbers are no longer as prominent. This makes it harder to read

Reply:
removed unclear field names in response struct refactor